seed/migrations/0197_pm_source_type.py

```diff
@@ -1,22 +1,18 @@
 from django.db import migrations
 
-from seed.models import ASSESSED_RAW, PORTFOLIO_RAW, ASSESSED_BS, PORTFOLIO_BS
+from seed.models import ASSESSED_BS, ASSESSED_RAW, PORTFOLIO_BS, PORTFOLIO_RAW
 
 
 def fix_source_type(apps, schema_editor):
+    # Fix PropertyState `source_type` fields that incorrectly list ESPM imports as Assessed imports
     pm_imports = apps.get_model('data_importer', 'ImportFile').objects.filter(source_program='PortfolioManager')
     property_states = apps.get_model('seed', 'PropertyState').objects.filter(
         import_file__in=pm_imports,
         source_type__isnull=False,
     )
 
-    for state in property_states.filter(source_type=ASSESSED_RAW):
-        state.source_type = PORTFOLIO_RAW
-        state.save()
-
-    for state in property_states.filter(source_type=ASSESSED_BS):
-        state.source_type = PORTFOLIO_BS
-        state.save()
+    property_states.filter(source_type=ASSESSED_RAW).update(source_type=PORTFOLIO_RAW)
```

Comment:
ahh, much better... yeah, this will be mmmuuch faster.

Reply:
Yup.  Just had to confirm that we weren't reimplementing the `save` method on any of these models